src/main/groovy/org/pih/warehouse/inventory/QuantityPickedByLocationAndProductDto.groovy

```diff
@@ -0,0 +1,9 @@
+package org.pih.warehouse.inventory
+
+import org.pih.warehouse.core.Location
+
+class QuantityPickedByLocationAndProductDto {
+    Location binLocation
+    InventoryItem inventoryItem
+    Integer quantityAllocated
```

Comment:
I made no empty lines here, to provoke the reaction :laughing: 
Joking, just for the another dto, without empty lines it looked weird to me, and since there were only 3 properties and let it go, but I would also love to decide on the convention.